quadriga/Docker/tomcat/Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM tomcat:9.0.95-jdk17
+
+
+ENV JAVA_OPTS="-DappConfigFile=file:/jdk17-path/bin"
+ENV TOMCAT_VERSION 9.0.95
+ENV CATALINA_HOME /usr/local/tomcat
+
+RUN mkdir app
+RUN mkdir data
+RUN mkdir data/config
+RUN mkdir data/files
+
+WORKDIR app
+
+RUN apt clean && \
+    rm -rf /var/lib/apt/lists/*
+
+RUN apt-get update && apt-get install git -y && apt install maven -y
+
+RUN git clone -b story/Q20-25 https://github.com/diging/quadriga-2.0.git
```

Comment:
We don't want to clone the code into the container but rather have a shared volume where the code lives and then deploy the app through tomcat (or copy over the war or something).